lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_lint_localizations.rb

```diff
@@ -22,17 +32,38 @@ def self.run_linter(params)
           install_path: resolve_path(params[:install_path]),
           version: params[:version]
         )
-        all_violations = helper.run(
+
+        helper.run(
           input_dir: resolve_path(params[:input_dir]),
           base_lang: params[:base_lang],
           only_langs: params[:only_langs]
         )
+      end
+
+      def self.report(violations:, base_lang:)
+        violations.each do |lang, lang_violations|
+          UI.error "Inconsistencies found between '#{base_lang}' and '#{lang}':\n\n#{lang_violations.join("\n")}\n"
```

Comment:
btw even if not directly related to this PR, I was thinking it could be nice to add `" - "` prefixes to each item in the list of `lang_violations`, to make it look more like a bullet-point list, and make it more readable for people using narrower Terminal windows where some long error messages could wrap to next lines…

```suggestion
          violations_list = lang_violations.map { |line| " - #{line}\n" }.join
          UI.error "Inconsistencies found between '#{base_lang}' and '#{lang}':\n\n#{violations_list}\n"
```